internal/worker.go

```diff
@@ -163,6 +164,10 @@ type (
 		// Optional: worker graceful shutdown timeout
 		// default: 0s
 		WorkerStopTimeout time.Duration
+
+		// Optional: Sets ContextPropagators that allows users to control the context information passed through a workflow
+		// default: no ContextPropagators
+		ContextPropagators []ContextPropagator
```

Comment:
may be make the jaeger or open tracing implementation (if exists) a default?

Reply:
Will do that in the next change when I actually add the tracing support. Enabling tracing will add the default tracing context propagator.